provision-new-dev-env.sh

```diff
@@ -9,14 +9,11 @@ title "Add upstream remote"
 git remote add upstream git@github.com:pykonik/pykonik.org.git
 git fetch upstream
 
-title "Create virtualenv"
-virtualenv -p python3 .meetpy-env
-source ./.meetpy-env/bin/activate
+title "Create virtualenv via pipenv"
+pipenv --python 3.7
```

Comment:
This shouldn't be there. `pipenv install` below will create new virtualenv with correct python version on its own.